apps/codeforafrica/src/components/AccoladeBadge/AccoladeBadge.js

```diff
@@ -0,0 +1,81 @@
+import { Typography, Grid, Box } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import badgeIcon from "@/codeforafrica/assets/badge.svg?url";
+
+const AccoladeBadgeRoot = styled(Grid, {
+  slot: "Root",
+})(({ theme }) => ({
+  background: theme.palette.background.main,
+  border: 0,
+  margin: "16px",
```

Comment:
`Grid` uses `margin` for aligning items so setting margin on grid can cause all sorts of issues.